code/src/components/Album.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+const Album = (props) => {
+
+
+  return (
+    <div className="album">
+      <a href={props.albumContent.external_urls.spotify}>
+        <div className="cover-wrapper">
+          <img className="cover" src={props.albumContent.images[1].url} alt="cover" />
+          <div className="icons">
+            <img className="icon favorite" src={require("./heart.svg")} alt="favorite icon" />
+            <img className="icon play" src={require("./play.svg")} alt="play icon" />
+            <img className="icon dots" src={require("./dots.svg")} alt="dots icon" />
+          </div>
```

Comment:
Your image should be still darkened when you hover over any of the icons. (you can experiment with opacities of backgrounds of every layer)

Your icons are not always properly centred: 

![image](https://user-images.githubusercontent.com/50926159/111640688-b863f900-87fc-11eb-8e0d-8c2be288f3da.png)

Reply:
Changes made